tests/e2e/api/coin.test.ts

```diff
@@ -0,0 +1,76 @@
+import { AptosConfig, Network, Aptos, Account, Deserializer } from "../../../src";
+import { waitForTransaction } from "../../../src/internal/transaction";
+import { RawTransaction, TransactionPayloadEntryFunction } from "../../../src/transactions/instances";
+import { TypeTagStruct } from "../../../src/transactions/typeTag/typeTag";
+import { SigningScheme } from "../../../src/types";
+
+describe("coin", () => {
+  test("it generates a transfer coin transaction with AptosCoin coin type", async () => {
+    const config = new AptosConfig({ network: Network.DEVNET });
+    const aptos = new Aptos(config);
+    const sender = Account.generate({ scheme: SigningScheme.Ed25519 });
+    const recipient = Account.generate({ scheme: SigningScheme.Ed25519 });
+    await aptos.fundAccount({ accountAddress: sender.accountAddress.toString(), amount: 100000000 });
```

Comment:
I suggest we make these numbers constants for the test

Reply:
yes, we need a test framework. will work on it after we have full test cover to our code (not a P0 for alpha version)